eng/DotNetBuild.props

```diff
@@ -38,6 +38,19 @@
     <ShortStack Condition="'$(DotNetBuildMonoEnableLLVM)' == 'true' or '$(DotNetBuildMonoAOTEnableLLVM)' == 'true'">true</ShortStack>
   </PropertyGroup>
 
+  <!--
+    Allow the VMR orchestrator to control whether or not to build rid-specific artifacts,
+    but provide defaults until the VMR orchestrator provides controls in all scenarios.
+  -->
+  <PropertyGroup Condition="'$(EnableDefaultRidSpecificArtifacts)' == ''">
+    <!-- Source-build always needs all artifacts to be published. -->
+    <EnableDefaultRidSpecificArtifacts Condition="'$(DotNetBuildSourceOnly)' != ''">false</EnableDefaultRidSpecificArtifacts>
+    <!-- Short-stack builds should always only publish RID-specific artifacts. -->
+    <EnableDefaultRidSpecificArtifacts Condition="'$(ShortStack)' == 'true'">true</EnableDefaultRidSpecificArtifacts>
+    <!-- If no override is specified, don't use RID-specific publishing. Instead, publish everything. -->
+    <EnableDefaultRidSpecificArtifacts Condition="'$(EnableDefaultRidSpecificArtifacts)' == ''">false</EnableDefaultRidSpecificArtifacts>
+  </PropertyGroup>
```

Comment:
These settings make sense to me but wouldn't they better fit into the VMR orchestrator as they aren't specific to runtime?

Reply:
Yes, they do fit better in the VMR. However, we need to put them here first so they don't break the VMR until I change the VMR to pass these options down (also in PR)

Reply:
Once the VMR controls are in, I'll come back and remove these.

Reply:
Consider filing a tracking issue and linking to it. Also, so that others can follow along and understand the potential for improvement in these code paths.